be/src/runtime/stream_load/load_stream_mgr.h

```diff
@@ -57,6 +57,14 @@ class LoadStreamMgr {
     }
     ~LoadStreamMgr() = default;
 
+    void clear() {
```

Comment:
I think close is a better name

Reply:
close() means wait_for_finish(), clear() will cancel without wait

Reply:
All clear() modify to close() ?

Reply:
Good idea, clear() means can be reuse, close() means finish and exit

Reply:
Fixed